tests/store/tracking/test_sqlalchemy_store.py

```diff
@@ -1943,24 +1981,19 @@ def test_sqlalchemy_store_can_be_initialized_when_default_experiment_has_been_de
     SqlAlchemyStore(db_uri, ARTIFACT_URI)
 
 
-class TestSqlAlchemyStoreSqliteMigratedDB(TestSqlAlchemyStoreSqlite):
+class TestSqlAlchemyStoreMigratedDB(TestSqlAlchemyStore):
     """
     Test case where user has an existing DB with schema generated before MLflow 1.0,
     then migrates their DB.
     """
 
     def setUp(self):
-        fd, self.temp_dbfile = tempfile.mkstemp()
-        os.close(fd)
-        self.db_url = "%s%s" % (DB_URI, self.temp_dbfile)
+        super()._setup_db_uri()
         engine = sqlalchemy.create_engine(self.db_url)
         InitialBase.metadata.create_all(engine)
         invoke_cli_runner(mlflow.db.commands, ["upgrade", self.db_url])
         self.store = SqlAlchemyStore(self.db_url, ARTIFACT_URI)
 
-    def tearDown(self):
-        os.remove(self.temp_dbfile)
```

Comment:
We don't need to override the tearDown method.

Reply:
Was this originally for Windows?

Reply:
I'm not sure, `TestSqlAlchemyStore.tearDown` also contains `os.remove(self.temp_dbfile)`.